packages/api/core/src/api/package.ts

```diff
@@ -269,6 +272,12 @@ export const listrPackage = ({
 
           type PackagerArch = Exclude<ForgeArch, 'arm'>;
 
+          const beforeAsarHooks = resolveHooks(forgeConfig.packagerConfig.beforeAsar, ctx.dir);
+          const afterAsarHooks = resolveHooks(forgeConfig.packagerConfig.afterAsar, ctx.dir);
+
+          const beforeCopyExtraResources = resolveHooks(forgeConfig.packagerConfig.beforeCopyExtraResources, ctx.dir);
+          const afterCopyExtraResources = resolveHooks(forgeConfig.packagerConfig.afterCopyExtraResources, ctx.dir);
```

Comment:
We don't need these hooks specified here, these are passed through with the spread operator on line 285

Reply:
I completely overlooked that line! Since _all other_ Packager hooks were wrapped in `sequentialHooks` / `resolveHooks`, I just assumed _every hook_ should, but those hooks only needed the extra code because they happened to be used internally by Forge.